framework/src/main/java/com/musala/sdcs/http/DeviceController.java

```diff
@@ -0,0 +1,39 @@
+package com.musala.sdcs.http;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.musala.sdcs.device.Device;
+import com.musala.sdcs.repository.DeviceRepository;
+
+@RestController
```

Comment:
Please add JavaDoc for the class and methods - annotations and request mapping are not explanatory enough.